spec/requests/api_registrations_spec.rb

```diff
@@ -452,17 +452,153 @@
     end
   end
 
+  def stub_successful_stripe_payment_intent(amount, currency)
```

Comment:
Why do you need this here and how does this affect existing Stripe tests?

Reply:
When testing the creation of payment tickets, we call the Stripe API which makes the tests significantly slower. I've included a test on line 526 which does query against the StripeAPI, to make sure that we are still constructing the payload appropriately, but for everything else (ie where behaviour of the Stripe API is not under test) I use `stub_successful_stripe_payment_intent` to mock the Stripe API instead of making live calls against it

Reply:
Removed the Stripe stubbing